java/CWE-326/Base64Encryption.ql

```diff
@@ -28,18 +28,20 @@ class Base64Sinks extends DataFlow::Node {
   }
 }
 
-class Base64EncryptionConfig extends TaintTracking::Configuration {
-  Base64EncryptionConfig() { this = "Base64EncryptionConfig" }
+module Base64EncryptionUsage implements DataFlow::ConfigSig {
```

Comment:
For consistency, current convention in the CodeQL repo is for `DataFlow::ConfigSig` modules to be named with the suffix `Config` and `Global` modules to end with `Flow`. 
https://github.com/github/codeql/blob/main/ql/ql/src/queries/style/DataFlowConfigModuleNaming.ql

```suggestion
module Base64EncryptionUsageConfig implements DataFlow::ConfigSig {
```

Reply:
Ah! Thank you - will make that update.